Misc/ACKS

```diff
@@ -2077,7 +2079,5 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
-Vlad Emelianov
-Andrey Doroschenko
```

Comment:
Andrey was removed here but not re-added

Reply:
That's because Andrey is already present at the correct alphabetically-ordered spot in the list. This is just removing a duplicate.

Reply:
He's already present in the right order:

https://github.com/python/cpython/blob/c32abf1f21c4bd32abcefe4d601611b152568961/Misc/ACKS#L465

xref:
* GH-18639
* GH-22277

Reply:
I DEMAND HIM TO BE PRESENT TWICE

okay, okay, j/k j/k